packages/mobile_app/lib/components/prophecy_record.dart

```diff
@@ -11,51 +13,99 @@ class ProphecyRecord extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final changesCount = prophecy.changes.length;
+
     double value = prophecy.model.value ?? 0.0;
+
     for (ImpactModel change in prophecy.changes) value += change.value;
```

Comment:
Would be nice to have some comments added what the calculations are doing (its more/less clear that they are adopting on changes count, but some detail wont hurt)

Reply:
I thought it is obvious, but I'll add comment.

Reply:
Done.